app/controllers/state_file/questions/submission_confirmation_controller.rb

```diff
@@ -0,0 +1,13 @@
+module StateFile
+  module Questions
+    class SubmissionConfirmationController < QuestionsController
+      def edit; end
+
+      private
+
+      def form_class
+        NullForm
+      end
+    end
+  end
+end
```

Comment:
I'm a little unclear of what happens after this screen to be able to update the feature specs.